boards/espressif/esp_wrover_kit/esp_wrover_kit_procpu.dts

```diff
@@ -102,6 +102,12 @@
 	pinctrl-names = "default";
 };
 
+&uart1 {
+	current-speed = <115200>;
+	pinctrl-0 = <&uart1_default>;
+	pinctrl-names = "default";
+};
```

Comment:
Missing `status="okay"` or is it on purpose?

Reply:
Since this is universal dev. board I intentionally didn't enable uart1 here.

Reply:
@uLipe but the uart1 node is disabled by default in esp32_common.dtsi. Do you think that it should be disabled more explicitly? :)

Reply:
From a kernel developer I would agree with you, we know by default nodes are disabled when the okay is not explicitly shown.

But from a user perspective, in general people that are using Zephyr for their first time, showing a peripheral node into the DTS without telling "hey this peripheral is supported, but it is disable currently" may lead them to get stuck because they think the peripheral is enabled, leading them to wast a considerable time to understand that the peripheral is actually present but needs to be enabled accordingly. 

Is that makes sense?